packages/composites/proof-of-reserves/src/utils/addressValidator.ts

```diff
@@ -75,6 +77,15 @@ export const validateAddresses = (
       case 'avalanche-fuji':
         validatedAddress = getValidAvalancheAddress(id, address)
         break
+      case 'multi-evm':
```

Comment:
Should we validate contract address as well?

Reply:
reasonable